dotCMS/src/main/webapp/html/js/dotcms/dijit/FileBrowserDialog.jsp

```diff
@@ -52,14 +59,9 @@
                                 <tbody dojoAttachPoint="detailsTableBody">
                                 </tbody>
                             </table>
-                            <table dojoAttachPoint="listTable" style="display: none;" class="listingTable">
-                                <tbody dojoAttachPoint="listTableBody">
-                                </tbody>
-                            </table>
-                            <table dojoAttachPoint="thumbnailsTable" style="display: none;" class="thumbnailTD" style="width:100%;">
-                                <tbody dojoAttachPoint="thumbnailsTableBody">
-                                </tbody>
-                            </table>
+
+                            <div dojoAttachPoint="thumbnailsTable" style="display: none" class="file-selector-tree__card-view">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)